### ISSUE TITLE
Default `WolkenpumpeMain` doesn't work because we want genNumChannels > 0

### ISSUE THREAD

**Issue by Sciss** [Sciss/Wolkenpumpe#19]
In GitLab by @Sciss on Jun 17, 2020, 01:00

```scala
Exception in thread "main" java.lang.IllegalArgumentException: requirement failed
	at scala.Predef$.require(Predef.scala:327)
	at de.sciss.nuages.ScissProcs$.applyWithActions(ScissProcs.scala:1024)
	at de.sciss.nuages.ScissProcs$.apply(ScissProcs.scala:178)
	at de.sciss.nuages.WolkenpumpeMain.registerProcesses(WolkenpumpeMain.scala:68)
	at de.sciss.nuages.WolkenpumpeMain.$anonfun$run$7(WolkenpumpeMain.scala:100)
	at de.sciss.nuages.WolkenpumpeMain.$anonfun$run$7$adapted(WolkenpumpeMain.scala:96)
```
It's a bit silly we can't use the class without overriding defaults.

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jul 19, 2020, 19:26

mentioned in commit f49ab554d2a25f74b2958993e991ec22f1ec7548

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jul 22, 2020, 02:40

closed via commit f49ab554d2a25f74b2958993e991ec22f1ec7548
